web/src/Pages/TopicPage/TopicCard/TopicCard.js

```diff
@@ -29,3 +31,13 @@ export default function TopicCard({ topic }) {
     </div>
   );
 }
+
+TopicCard.propTypes = {
+  topic: PropType.shape({
+    name: PropType.string,
+    description: PropType.arrayOf(PropType.object),
```

Comment:
This code works on my machine. But do try it on yours and if you see any errors in the console, revert back to `PropType.object` as @DomVinyard has said it's fine.
```suggestion
    description: PropType.arrayOf(PropType.shape({
        audience: PropType.number.isRequired,
        long: PropType.string.isRequired,
        short: PropType.string.isRequired,
      }).isRequired,),
```